docs/getting_started/basics/authoring.md

```diff
@@ -1,36 +1,44 @@
 # Workflow Authoring
 
-By opening Shift you will be prompted to an empty board.  
+This page is dedicated to explain the most basic steps for creating a Shift workflow. 
+When launching Shift, an new and empty board will be opened where nodes and connections can be added to start building the workflow. 
 
-## Creating Your First Node
+## Creating a Node
 
-To create a new [operator](../intro/terminology.md/#operator), you can either press the `Tab Key` or `Double Left-Click` with your mouse.  
+To create a new [node](../terminology.md/#operator), press the `Tab` key or `Double Left-Click` to open the "*Create Node*" dialog. This dialog enables searching for nodes by name and/or filtering by [catalog](../terminology.md/#catalog). The nodes will be listed alphabetically underneath, where they can be added to the Shift board by double-clicking on them.
 
-![Create Node Dialog](../images/node_creation_dialog.png "Create Node Dialog")  
+![Create Node Dialog](../../images/create_node_dialog.gif "Create Node Dialog")   
 
-Then you can choose the type of operator you want to create and double-click on its name to create it.
+Alternatively, nodes can be added to the board using the [Nodes List widget](../ui_overview.md/#operator) present in the Shift interface.
 
 ## Creating Connections
 
-If you want a certain operator to pass information to another one, you can connect their [plugs](../intro/terminology.md/#plug). To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  
+To establish a connection between plugs from different nodes, `Left-Click` and `Drag` from one plug to the desired plug. By creating a connection, the workflow can propagate the data produced in the output plugs of a node and transfer it to the connected input plug of another node.
 
-![Connected Nodes](../images/connected_nodes.png "Connected Nodes")  
-
-This will have created the simplest example of a Shift workflow.
+![Connected Nodes](../../images/connect_plugs.gif "Connected Nodes")  
 
 ## Creating New Plugs
 
-Certain nodes in Shift allow the creation of new plugs. To do that you can `Right-Click` on the node and select the "*Create New Plug*" option if available. It will prompt you to a new dialog asking to fill in the information needed to create the plug.
+Certain nodes in Shift allow the creation of new plugs. To create a new plug, `Right-Click` on the node to open its context menu and select the "*Create New Plug*" option if available. This will open the "*Add Plug*" dialog, which contains all the information fields required to create a custom plug. These are:
+
+- **Name**: The name of the plug. This must be unique among the plugs of the same direction in the same node. For example, if an input plug named *value* already exists in the node, no other input plug can have that name.
```

Comment:
I just realized of something. I don´t know if it´s an issue to solve or a limitation to document.
Right now when we create a plug, the name must be unique, not only in the selected direction, but in the node. If the plug already exists in the oposite direction, the UI doesn´t allow to create the plug, this is only allowed using the through direction when creating the first one.
![image](https://github.com/user-attachments/assets/a724f9c3-edd4-42d6-910b-e87eb3a869b2)

So:

1. Should we create an Issue to solve this limitation?
2. Should we document the limitation because it's not an Issue?

Reply:
Hmm I think this was not like this before, but I might be wrong. As far as I know input and output nodes can have the same name.

Reply:
I think that this current limitation can be solve checking if the plug exist only for the selected direction, because probably we are filtering all the plugs. So is something that we could solve. Then it's an Issue right? If you want I can open a Issue and probably solve it, should be easy. And we leave the documentation like it is.

Reply:
We should bring  this into a meeting.

Reply:
I think we have agree on today's scrum to create an Issue to change this behavior. As soon as we agree that the change is good to go I can approve this PR like the documentation is ok then. Let me know @juliavisserinbibo when the Issue is create, or something and I will approve

Reply:
The issue has been created 
https://github.com/Inbibo/shift/issues/1104